common/test/acceptance/tests/studio/test_studio_tabs.py

```diff
@@ -0,0 +1,110 @@
+            'Static tab should not be present on the page for a newly created course'
+        )
+        self.pages_page.add_static_page()
+        self.assertTrue(
+            self.pages_page.is_static_page_present(),
+            'Static tab should be present on the page'
+        )
+
+    def test_user_can_delete_static_tab(self):
+        """
+        Scenario: Users can delete static pages
+            Given I have created a static page
+            When I "delete" the static page
+                Then I am shown a prompt
+            When I confirm the prompt
+                Then I should not see any static pages
+        """
+        self.assertFalse(
+            self.pages_page.is_static_page_present(),
+            'Static tab should not be present on the page for a newly created course'
+        )
+        self.pages_page.add_static_page()
+        self.pages_page.delete_static_tab()
+        self.assertFalse(
+            self.pages_page.is_static_page_present(),
+            'Static tab should not be present on the page after the deletion'
+        )
+
+    def test_user_can_edit_static_tab(self):
+        """
+        Scenario: Users can edit static pages
+            Given I have created a static page
+            When I "edit" the static page
+            And I change the name to "New"
+                Then I should see static page named "New"
+        """
+        self.assertFalse(
+            self.pages_page.is_static_page_present(),
+            'Static tab should not be present on the page for a newly created course'
+        )
+        self.pages_page.add_static_page()
+        self.assertNotEqual(self.pages_page.static_tab_titles[0], "New")
+        self.pages_page.click_edit_static_page()
+        self.pages_page.open_settings_tab()
+        self.pages_page.set_field_val("Display Name", "New")
+        self.pages_page.save()
+        self.pages_page.refresh_and_wait_for_load()
+        self.assertEqual(self.pages_page.static_tab_titles[0], "New", "The title of the tab is not updated")
+
+    def test_user_can_reorder_static_tabs(self):
+        """
+        Scenario: Users can reorder static pages
+            Given I have created two different static pages
+            When I drag the first static page to the last
+                Then the static pages are switched
+            And I reload the page
+                Then the static pages are switched
+        """
+        self.assertFalse(
+            self.pages_page.is_static_page_present(),
+            'Static tab should not be present on the page for a newly created course'
+        )
+        self.pages_page.add_static_page()
+        self.pages_page.click_edit_static_page()
+        self.pages_page.set_field_val("Display Name", "First")
+        self.pages_page.save()
+        self.pages_page.add_static_page()
+        self.pages_page.drag_and_drop_first_static_page_to_last()
+        self.pages_page.refresh_and_wait_for_load()
```

Comment:
same as above comment...why reload the page?

Reply:
In lettuce, there was a page refresh required, that is why this step was done here.